.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,40 @@
+# Pull Request Template
+
+## Summary
+
+* [Briefly describe the changes made in this PR.]
+* [Include any relevant context, such as the issue or feature being addressed.]
+
+## Changes Made
+
+* [List specific files or directories affected by these changes]
+* [Describe any significant updates, rewrites, or new code added]
+* [Mention any removed or deleted files or code]
+
+## Relevant Details
+
+* **Affected Code**: [List specific code paths, functions, or classes changed or updated]
+* **Impact**: [Explain how the changes affect the project's functionality, performance, or security]
+* **New Features/Changes**: [Describe new features or significant changes added in this PR]
+* **Fixed Issues**: [List specific issues or bugs fixed by these changes]
+
+## Verification
+
+To verify this PR, you can:
+
+* Run automated tests or scripts to ensure the changes do not introduce errors
+* Review code for style, security, and best practices
+* Verify that all changed files are properly formatted and consistent
+
```

Comment:
Replace with actual test commands, such as `./tests/test-backend-ops` and reference to [ggml-ci](https://github.com/ggerganov/llama.cpp/tree/master/ci)